### ISSUE TITLE
Metadata contact lang attribute problem

### ISSUE THREAD

**Issue by stevenao** [elerch/SAML2#15]
Testshib and some other idp are complaining about the generated metadata that the contact elements don't have a lang attribute. I think it is better to have a way a way to specific that as an option. See my commit: https://github.com/stevenao/SAML2/commit/4ec93982d462d919f748d9d6bc76a53b7572cd0d.

**Comment by svrooij** [comment]
I agree with your comment. Maybe you can create a pull request for it? And make it optional?
